lib/shopify-cli/oauth.rb

```diff
@@ -64,7 +65,7 @@ def server
 
     private
 
-    def initiate_authentication(url)
+    def authentication_uri(url)
```

Comment:
I wonder if the name of this should remain `initiate_authentication`?  `authentication_uri` doesn't really convey that this method is initiating a server thread, which is its primary purpose.